media-types.md

```diff
@@ -3,6 +3,7 @@
 The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
 
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
+- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
```

Comment:
On Tue, Aug 30, 2016 at 01:48:59PM -0700, Stephen Day wrote:
> > @@ -3,6 +3,7 @@
> >  The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
> >  
> >  - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
> > +- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
>
> @wking Types of objects are not detected. The type of an object
> comes from trusted descriptors or a metadata storage system.

If you have a descriptor holding type
application/vnd.oci.descriptor.foreign.v1+json, what does it point to?
Does it point to another descriptor, and that descriptor has a type
like application/vnd.oci.image.serialization.rootfs.tar.gzip?  That
would be the first case I describe in [1].  The current Docker
approach, as I understand it, is the third case I describe in [1].  Or
are you intending this type be used in a way that I don't describe in
[1] at all?

[1]: https://github.com/opencontainers/image-spec/pull/216#issuecomment-243285594